src/app/api/scrim/update-scrim-options/route.ts

```diff
@@ -11,6 +11,7 @@ type UpdateScrimBody = {
   scrimId: number;
   date: string;
   guestMode: boolean;
+  maps: Array<{ id: number; replayCode: string }>;
```

Comment:
Minor nitpick but I prefer using `obj[]` over the `Array<obj>` generic for consistency. There are a few places in the codebase where we use Array due to not wanting to use parens but this is not one of those places. I would change this to prefer the `[]` style.